Drv/TcpServer/test/ut/TcpServerTester.cpp

```diff
@@ -87,15 +87,16 @@ void TcpServerTester ::test_with_loop(U32 iterations, bool recv_thread) {
                 while (not m_spinner) {}
             }
         }
-        client.close(); // Client must be closed first or the server risks binding to an existing address
+
         // Properly stop the client on the last iteration
-        if ((1 + i) == iterations && recv_thread) {
-            this->component.shutdown();
```

Comment:
@LeStarch Can you confirm that we do not need this `shutdown`, since it's already part of the `stop` function?

Keeping it is causing a timeout on my setup (the UT tasking more than 1s)